### PR TITLE
Include number of mirror tiles in guessing keys to be able to distinguish between MAGIC 1 and MAGIC 2

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

**Review by kosack — DISMISSED** [review]
let's hope eventually we can just use the new metadata in simtel to avoid guessing altogether!

**Comment by kosack** [comment]
You really need to add some descriptions to these though - took me a bit to figure out what M1 and M2 are

**Comment by maxnoe** [comment]
Sorry, I wanted to open this as draft...

**Review by kosack — APPROVED** [review]

**Review by kosack — APPROVED** [review]

File: ctapipe/instrument/guess.py [hunk 1]
```diff
@@ -9,32 +9,63 @@
...
     # Non-CTA Telescopes
-    GuessingKey(1039, 16.97): GuessingResult("LST", "MAGIC", "MAGICCam", 1),
-    GuessingKey(960, 15.0): GuessingResult("MST", "HESS-I", "HESS-I", 1),
-    GuessingKey(2048, 36.0): GuessingResult("LST", "HESS-II", "HESS-II", 1),
-    GuessingKey(1440, 4.89): GuessingResult("SST", "FACT", "FACT", 1),
-}
+    (GuessingKey(1039, 16.97, 964), GuessingResult("LST", "MAGIC-1", "MAGICCam", 1)),
+    (GuessingKey(1039, 16.97, 247), GuessingResult("LST", "MAGIC-2", "MAGICCam", 1)),
+    (GuessingKey(1039, 17.0, 964), GuessingResult("LST", "MAGIC-1", "MAGICCam", 1)),
+    (GuessingKey(1039, 17.0, 247), GuessingResult("LST", "MAGIC-2", "MAGICCam", 1)),
```
[the remark this answers is not in the document]
Comment (maxnoe) [reply ctapipe/instrument/guess.py:60]: Older (and possibly newer) simulations were produced with 16.97 m focal length, the current config with 17 m, to support both cases, both have to be included in the guessing keys.

**Review by nbiederbeck — APPROVED** [review]

**Pull request merged**